app/src/main/java/edu/uc/groupproject/covid19tracker/service/CasesByCountryDataProvider.kt

```diff
@@ -10,8 +10,8 @@ class CasesByCountryDataProvider {
     /**
      * Build http API request to retrieve covid19 cases data by country
      */
-    var client = OkHttpClient()
-    var request = Request.Builder()
+    private var client = OkHttpClient()
+    private var request: Request = Request.Builder()
```

Comment:
Added visibility modifier per documentation: https://kotlinlang.org/docs/reference/visibility-modifiers.html

Reply:
That's a good idea... though the type Request can be inferred, so that is probably not needed.